engineapi/alembic/versions/040f2dfde5a5_request_id_bytes_column.py

```diff
@@ -0,0 +1,75 @@
+"""Request ID bytes column
+
+Revision ID: 040f2dfde5a5
+Revises: b4257b10daaf
+Create Date: 2023-08-10 08:58:22.052336
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '040f2dfde5a5'
+down_revision = 'b4257b10daaf'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('call_requests', sa.Column('request_id', sa.BigInteger(), nullable=True))
+    op.create_index(op.f('ix_call_requests_request_id'), 'call_requests', ['request_id'], unique=False)
+    op.create_unique_constraint(op.f('uq_call_requests_registered_contract_id'), 'call_requests', ['registered_contract_id', 'request_id'])
+
+    # Manual
+    # Fetch IDs of duplicates for 'dropper-v0.2.0' call_request_type and delete it
+    op.execute("""WITH Duplicates AS (
+    SELECT 
+        id,
+        registered_contract_id,
+        call_request_type_name,
+        parameters->'requestID' AS request_id,
+        created_at,
+        ROW_NUMBER() OVER (
+            PARTITION BY 
+                registered_contract_id,
+                call_request_type_name,
+                parameters->'requestID' 
+            ORDER BY created_at ASC
+        ) AS row_num
+    FROM call_requests
+    WHERE call_request_type_name = 'dropper-v0.2.0' 
+),
+DeleteDuplicates AS (
+SELECT id
+FROM 
+    Duplicates
+WHERE
+    row_num < (
+        SELECT COUNT(*) FROM Duplicates d2
+        WHERE d2.registered_contract_id = Duplicates.registered_contract_id
+        AND d2.call_request_type_name = Duplicates.call_request_type_name
+        AND d2.request_id = Duplicates.request_id
+    )
+)
+DELETE FROM call_requests WHERE id IN (SELECT id FROM DeleteDuplicates);""")
+               
+    # Fulfill not empty requestID values
+    op.execute("UPDATE call_requests SET request_id = CAST(parameters->>'requestID' AS bigint) WHERE parameters->>'requestID' IS NOT NULL;")
```

Comment:
Not `bigint` but `decimal`.

`bigint` isn't large enough to hold `uint256` - which is what most of our request IDs are.